doc/pages/source/documentation/index.md

```diff
@@ -950,7 +950,24 @@ to the chain of middleware execution.
 ### Sub-plans
 
 -   *when to use?*
--   *how to use?*
+
+To use sub-plans, you must include the `Dynflow::Action::WithSubPlans` module
+and override the `create_sub_plans` method. Inside the `create_sub_plans`
+method, you use the `trigger` method to create sub-tasks that will be executed
+in sequence but will not block the parent task.
```

Comment:
So, if you trigger two sub-tasks, in the same `create_sub_plans` method, the second task isn't guaranteed to start after the first task is finished?